Homework 03 Chore: Added Jerimiah's Quiz

Dlafferty251 opened the pull request:
Hello this is my pr regarding davids addition to lesson 03

anthonydmays reviewed (changes requested):
This PR should not include any `lesson_01` work. Also, you should only have your changes in this PR, not David's. It looks like you can just delete anything that isn't your changes and that would get this PR in better shape.

Dlafferty251 commented:
I deleted both Khayla and Meikos HTML files I guess they showed up because when I pulled to get davids quiz it also pulled that I'm sorry I really think Im good I did npm run check and made sure no extra files were added i mean it says I deleted meikos file but that should be ok right??

Dlafferty251 reviewed (commented):
Changing Up Some files to make merge check work

Dlafferty251 commented:
I have added Khaylas Quiz to the repo

Dlafferty251 reviewed (commented):
I Have Reviewed changes the files being changed are me deleting lesson_01 work

Dlafferty251 commented:
I have added jasons quiz

anthonydmays reviewed (approved):
LGTM
